### PR TITLE
Update telemetry for 2.0 multi-tab

### LINKED ISSUE #1509: Rocket 2.0 data/telemetry
After spec lock down we should figure out what telemetry are needed and how these should be embed in product.  Data review again will be necessary.

### PR DESCRIPTION
**Pull request by idiotmax**
Closes #1509

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java [hunk 1]
```diff
@@ -332,6 +337,12 @@ public static void copyImageEvent() {
         TelemetryEvent.create(Category.ACTION, Method.COPY, Object.BROWSER_CONTEXTMENU, Value.IMAGE).queue();
     }
 
+    public static void addNewTabFromContextMenu() {
+        TelemetryEvent.create(Category.ACTION, Method.ADD, Object.TAB, Value.LINK)
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: I'm uncertain here: to be comparable with shareLinkEvent() it should be
TelemetryEvent.create(Category.ACTION, Method.OPEN_TAB, Object.BROWSER_CONTEXTMENU, Value.LINK).queue();
but to be comparable with addNewTabFromHome() it should be
TelemetryEvent.create(Category.ACTION, Method.ADD, Object.TAB, Value.LINK).queue();
I think I'm favoring the first one?

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java [hunk 2]
```diff
@@ -453,12 +464,15 @@ public static void clickAddToHome() {
...
-        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.TOOLBAR, Value.CAPTURE).queue();
+        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.TOOLBAR, Value.CAPTURE)
+                .extra(Extra.VERSION, Integer.toString(2))
+                .queue();
     }
 
     public static void clickTopSiteOn(int index) {
         TelemetryEvent.create(Category.ACTION, Method.OPEN, Object.HOME, Value.LINK)
                 .extra(Extra.ON, Integer.toString(index))
+                .extra(Extra.VERSION, Integer.toString(2))
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: I'm actually uncertain why we upgrade this. (We have not changed it somehow?) Maybe we want to know if the new home / browser navigation has an impact but maybe we should upgrade more? Like Home / BrowserFragment first level item clicks?

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java (hunk 1 above)
Comment (idiotmax) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: For easier quantitative comparison, do we focus on how user add a tab(add,tab,xxx)? Or which one is more popular when user long click on ContextMenu([share/copy/save],browser_contextmenu,[image/link])?

Comment (mTwTm) [reply app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: That sounds like a product call but I think of UI pings more related to UI itself so comparing these are a different dash board which we select different pings from different UIs.

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java [hunk 3]
```diff
@@ -332,6 +338,12 @@ public static void copyImageEvent() {
         TelemetryEvent.create(Category.ACTION, Method.COPY, Object.BROWSER_CONTEXTMENU, Value.IMAGE).queue();
     }
 
+    public static void addNewTabFromContextMenu() {
+        TelemetryEvent.create(Category.ACTION, Method.ADD, Object.BROWSER_CONTEXTMENU, Value.LINK)
+                .extra(Extra.SOURCE, Object.BROWSER_CONTEXTMENU)
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: Duplicate Object.BROWSER_CONTEXTMENU?

File: app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java [hunk 4]
```diff
@@ -332,6 +338,10 @@ public static void copyImageEvent() {
         TelemetryEvent.create(Category.ACTION, Method.COPY, Object.BROWSER_CONTEXTMENU, Value.IMAGE).queue();
     }
 
+    public static void addNewTabFromContextMenu() {
+        TelemetryEvent.create(Category.ACTION, Method.ADD, Object.BROWSER_CONTEXTMENU, Value.LINK).queue();
```
Comment (mTwTm) [inline app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java]: It feels a bit weird to not see tab here but I'm ok with that.

**Pull request merged**